app/assets/stylesheets/spotlight/_sir-trevor_overrides.scss

```diff
@@ -17,6 +17,12 @@
   margin-bottom: 3*$padding-large-vertical;
 }
 
+.quote__content {
+  border-left: 0.4em solid $gray-lighter;
+  color: $gray;
```

Comment:
I think `$text-muted` is actually darker than `$gray-lighter` (#eee versus #777). So `$text-muted` might be fine (I was just going by the screenshot since I don't have an active Spotlight dev environment going at the moment).